{"uuid": "3b1a09df-f0b7-4f7d-b465-9ad382c098d4", "children": ["61ee8512-9256-4d69-a3b5-2da68455c637"], "befores": [{"name": "log_on_failure", "status": "passed", "start": 1716182694531, "stop": 1716182694531}], "afters": [{"name": "log_on_failure::0", "status": "broken", "statusDetails": {"message": "NameError: name 'driver' is not defined\n", "trace": "  File \"C:\\Users\\shirisha.mannem\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python Projects\\framework2\\selenium-python-framework\\env\\Lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\shirisha.mannem\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python Projects\\framework2\\selenium-python-framework\\env\\Lib\\site-packages\\_pytest\\fixtures.py\", line 893, in _teardown_yield_fixture\n    next(it)\n  File \"C:\\Users\\shirisha.mannem\\OneDrive - Feuji Software Solutions Pvt Ltd\\Documents\\Python Projects\\framework2\\selenium-python-framework\\tests\\conftest.py\", line 163, in log_on_failure\n    allure.attach(driver.get_screenshot_as_png(), name=\"failed_test\", attachment_type=allure.attachment_type.PNG)\n                  ^^^^^^\n"}, "start": 1716182704736, "stop": 1716182704736}], "start": 1716182694531, "stop": 1716182704737}